example/transform/huggingface_model_json.ipynb

```diff
@@ -141,38 +121,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "sample size of processed input data:  4\n"
+      "sample size of processed input data:  1000\n",
+      "Example uniflow context data:\n",
```

Comment:
Nit: too much output here, and the output content is a bit difficult to distinguish.